### PR TITLE
Supercarrier aliases missing AircraftAliases

### PR DESCRIPTION
**Pull request by emcek**
Looks like `SuperCarrier.json` is missing in `AircraftAliases.json`

### CODE REVIEW

**Comment by charliefoxtwo** [comment]
AircraftAliases.json is an auto-generated file, so if this were an issue we should probably address this at the source rather than here. However, I believe this is intentional as the supercarrier itself is not a playable module (just like the NS430).

That said, again like the NS430, I believe the supercarrier should appear in the export list for all playable modules, so perhaps that could be a problem.

Was there a specific issue you were encountering?

**Comment by emcek** [comment]
> AircraftAliases.json is an auto-generated file, so if this were an issue we should probably address this at the source rather than here. However, I believe this is intentional as the supercarrier itself is not a playable module (just like the NS430).
> 
> That said, again like the NS430, I believe the supercarrier should appear in the export list for all playable modules, so perhaps that could be a problem.
So, should I check codebase and try add `Supercarrier` as playable module?

> Was there a specific issue you were encountering?
Nothing special, I was checking `AircraftAliases.json` and this case looked odd to me.

**Comment by charliefoxtwo** [comment]
No, the supercarrier should not be added as a playable module (as it's not - you can't start a mission as a supercarrier pilot lol).

When a user is in a mission, they are using a single playable module. For example, if they start with the F18 on the supercarrier, their playable module would be the F18, and bios will export data for the F18 module addresses. Bios will also export MetadataStart, MetadataEnd, and CommonData module addresses, as they're common to all aircraft. We just also need to make sure the supercarrier is treated like CommonData basically, so that it's exported with all modules (though, I wonder what happens if a map has multiple supercarriers 🤔)

File: Scripts/DCS-BIOS/doc/json/AircraftAliases.json [hunk 1]
```diff
@@ -116,7 +116,7 @@
                  "Su-33": [ "CommonData", "FC3", "NS430" ],
                  "Su-57": [ "CommonData", "FC3" ],
         "Super_Etendard": [ "CommonData", "FC3" ],
-          "Supercarrier": [ "CommonData" ],
+          "Supercarrier": [ "CommonData", "SuperCarrier" ],
```
Comment (charliefoxtwo) [inline Scripts/DCS-BIOS/doc/json/AircraftAliases.json:119]: SuperCarrier should not be added here, as it's not a playable module

**Review by charliefoxtwo — CHANGES_REQUESTED** [review]
Marking this as request changes so we know not to merge in its current state (this file is autogenerated, and any changes would immediately be overwritten on starting a mission).

If any incorrect/buggy behavior is occurring, perhaps an issue should be opened to discuss further? Though I'll be honest, given the current state of the supercarrier and lack of any controls I don't know that it's worth spending too much time on at this stage.

**Comment by charliefoxtwo** [comment]
Looked into this a little further - the reason SuperCarrier isn't in AircraftAliases.lua is because it's not exported by default
https://github.com/DCS-Skunkworks/dcs-bios/blob/ec3bf2cf8872d2d4debd32169c24ddf1d714aaf3/Scripts/DCS-BIOS/BIOS.lua#L66

I believe there were issues with the exports last time we tried, so it has stayed disabled. We can revisit this though.

**Comment by charliefoxtwo** [comment]
I plan to revisit this once the next patch drops, as the planned addition of the air boss station may change the way the supercarrier operates with respect to dcs-bios.

**Comment by charliefoxtwo** [comment]
I'm going to go ahead and close this PR since to my knowledge there's no way to get data out of the Supercarrier currently, and as such the module should stay disabled. Hopefully in the future we'll be able to get the data from it though!

**Pull request closed without merging**